dll/win32/shell32/shlexec.cpp

```diff
@@ -593,6 +593,26 @@ static LPWSTR SHELL_BuildEnvW( const WCHAR *path )
     return new_env;
 }
 
+static LPWSTR SHELL_ExpandEnvironmentStrings(LPCWSTR psz)
```

Comment:
Isn't there already such a function in the shell? SHExpandEnvironmentStringsW

Reply:
Deleted.